### DISCUSSION TITLE
Can't login to my work account, phone # in use

### DISCUSSION THREAD

**Discussion by ODelibalta** [vercel/community#4854] [Account Issue]
### Question

Hello, 

I've been working on a personal project which was hosted at Vercel on my personal account. 

That personal project has become a work project. I moved it over to my work email account at Vercel. I remember being able to login and set everything up. That project was finished months ago and we have not touched it since. 

Now, when I try to login to my work email account at vercel, it is asking me to enter my cell number. I do enter. Then it says this phone number is already in use... 

I logged back into my personal account and that phone number is not entered anywhere... 

I searched through all my email accounts and try to login with any of them that received an email from Vercel in the past, they do not even work. Says the account does not exist. which is fine but then who entered my cell phone number? 

Why is this asking me for my phone number now? How can one use this platform if they have personal and a work account but a single phone number. It does not make sense to me. 

How can I login to my work email account now?

**Comment by swarnava** [comment]
Hi,

please email us at registration@vercel.com

**Reply by ODelibalta** [reply]
If you would then kindly respond to the email for this same issue I sent to that email address on Sept 27, I'd greatly appreciate it.
